.travis.yml

```diff
@@ -1,11 +1,16 @@
-language: c
+language: julia
+julia:
+  - 1.1
+  - nightly
+
 env:
  global:
    - DONT_FETCH_GAP_JULIA_PKG=yes
-   - JULIAROOT=$TRAVIS_BUILD_DIR/julia
+   - DONT_FETCH_JULIA=yes
    - GAPROOT=$TRAVIS_BUILD_DIR/gap
    - MAKEFLAGS="-j4"
    - JULIA_TRACK_COVERAGE=yes
+   - JULIA_PROJECT=${HOME}/.julia/environments/v1.0/
```

Comment:
Naive question, but: does using `v1.0` also work with the 1.1 and nightly builds? I am assuming it does, else you'd not have added it and Travis would not succeed, I am simply surprised... And I checked, on my system with Julia 1.1, there is only `~/.julia/environments/v1.1/` but no `~/.julia/environments/v1.0/`. So what gives?

Reply:
Actually, we could choose `/any/dir/that/is/not/${TRAVIS_BUILD_DIR}` here, because at the beginning (before the first `Pkg.add`) it just needs to be empty. I have choosen this one, but I can do a more custom one if you prefer.